mautic/mautic
# Change 'Email Received' to 'Email Read' in Segment Filters to reflect actual behavior

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | No
| Issues addressed (#s or URLs) | #1851
| BC breaks? | No
| Deprecations? | No

### Description:

The segment filter 'Email Received' actually filters contacts that have read the email.
This pull request changes that wording to reflect the actual behavior of the filter.

### Steps to test this PR:
1. Write an email 
2. Create a segment with an 'Email Read' filter targeting that email
3. Send the email to a contact
4. Check that this contact has NOT been added to the segment
5. After the contact has read (or just opened) the email check that he has
